x/bank/types/genesis.go

```diff
@@ -58,6 +58,13 @@ func NewGenesisState(params Params, balances []Balance, supply sdk.Coins, denomM
 	}
 }
 
+/*
+func DefaultMetadata() []Metadata {
+	defaultMetaData := Metadata{Description: sdk.DefaultBondDenom, Base: sdk.DefaultBondDenom, Display: sdk.DefaultBondDenom}
+	return []Metadata{defaultMetaData}
+}
+*/
```

Comment:
Does it make sense to add this function and use it in `DefaultGenesisState` instead of empty `Metadata` list? The `DefaultSupply` is empty, so probably we don't need anything. WDYT?